### DISCUSSION TITLE
[Design/Discussion] Evidence Storage

### DISCUSSION THREAD

**Discussion by matttheficus** [ParadiseSS13/Paradise#20978] [Documents]
# Issues with the Current System
* Currently, evidence is a joke on every single Map we have available on the server.
* Anecdotally, I have stolen upwards of 60+ TC worth of items from Evidence, and this was as a vampire during a VampTot round. I have seen a hijacker steal 120+ TC worth of gear and open uplinks before.
* All it takes is losing one officer to an/multiple Antagonists for Evidence to become wide open. This is on top of the looted Security gear, Brig/Armory access from the ID, and the SecComms.
* There are zero cameras, sometimes only one wall, and at MOST two to three doors to go through to get to Evidence.
* Evidence is a nightmare to manage. The Evidence Bags are seldom used due to being a bulky mess that nobody wants to sift through in a big ass pile stuffed into a locker.
* Vampires/Clings should not be able to basically become ClingTot/VampTots due to a simple walk into evidence and walk out.
* With Contractors, there's already a glut of gear out and available, and with the new Bundles, some very good discount bulk buys.

# IC Fixes for Some of the Issues
Currently, as Warden, the only way I've managed to secure the Evidence rooms is by installing a camera, replacing some of the lockers with spare Sec-access ones, and keeping High Value Class S (Chainsaws, 357s, etc) in the Armory if an Evidence breach has occurred. On top of this, Evidence Bags are completely useless as they only really clutter the lockers in the room itself.

# Proposed Changes
The main idea is to add a "Evidence Storage Safe" to the Evidence Locker Rooms of each Station. Unlike the Vault/Captain's Safe, this safe would have a PIN Pad for its code (which would be on a piece of paper in the Warden, Det, and HoS's bags, perhaps like an EFPOS paper).

## Interactions
- The Warden/HoS/Det can input the PIN to "open" the safe and allow for items to be removed at will. They would then click a button to "relock" it after.
- Anybody is able to place items INTO the Safe (think like a dropbox at a Library). Perhaps it can even add notes to the deposit so you can add names/investigation notes to it. Could even link it to the Sec Records Console so the Det has more stuff to sift through for investigations.
- You would NOT be able to place Theft Items (NAD/HoS Gun/etc) in the safe.
- You would NOT be able to place bags in the safe.
- Emaging the Safe would dispense ONE random item, and then go on a cooldown.
- EMPing the Safe would have a CHANCE to dispense ONE random item, on a longer cooldown.
- "Breaking" the Safe would be a lot like the current Crematorium, it doesn't break it FULLY (i.e. destroys it), but instead disables it until its repaired. That, or I can just make it invulnerable.

These would allow Antagonists to STILL raid Evidence for items but it comes at more of a risk. They either have to invest in the Holy Emag or they need to risk alerting Security with EMPs. This also prevents you from just waltzing in for free 20+ TC worth of items.

**Comment by Mitchs98** [comment]
Please, dear god, yes. Something like this or security having more leeway in deconning/destroying S-Class needs to happen. I am so totally fed up with Evidence being essentially a free all you can eat buffet for anyone that bebops their way into it. Let us not forget that all heads of staff have access to it as well, so they don't even need to steal from an officer or get an emag. They just waltz in at an opportune time and then take whatever they want.

**Comment by RagingLemons1** [comment]
what do you think about having safe destroying items periodically (one random item per like 5-10 minutes) for an increase to the sec budget?

I think emagging or EMPing the safe should dispense more than just one item (one to three?). Breaking into evidence can be riskier than breaking into the armory, which has a ton more weapons, due to the constant sec presence in processing on some maps, and higher risk should make for higher reward. Maybe put the safe in the armory.
